CuratedContent/SoftwareVersioning.md

```diff
@@ -0,0 +1,27 @@
+# Software Versioning
+<!--deck text start-->
+Most software changes over time.  An explicit versioning scheme is a key way to communicate to your users about those changes.
+<!--deck text end-->
+
+#### Contributed by [David E. Bernholdt](https://github.com/bernhold)
+
+#### Publication date: September 24, 2021
+
+Resource information | Details
+:--- | :--- 
+Wikipedia article  | [Software versioning](https://en.wikipedia.org/wiki/Software_versioning)
+Web site | [Semantic Versioning](https://semver.org/)
+Web site | [Calendar Versioning](https://calver.org/)
+
+Versioning is the process of assigning unique identifiers to instances of a product.  
```

Comment:
This newline shows up in both the GitHub rendering and the preview.bssw.io site as shown in the following screenshots:

![image](https://user-images.githubusercontent.com/1959736/134515782-f07b9e1a-dda4-4cba-9310-367e3cf012d8.png)

**On GitHub**

<br>

![image](https://user-images.githubusercontent.com/1959736/134515655-a899f28b-0bf6-4c23-bc39-fd0ac5e219d6.png)

**On preview.bssw.io**

<br>


It think we want to remove this newline.